recommended/gitconfig-prod-mac-citrix

```diff
@@ -0,0 +1,20 @@
+[color]
+	ui = auto
+[core]
+	autocrlf = false
+	editor = nano
+	eol = lf
+[credential]
+	helper = cache --timeout=86400
+[diff "ipynb"]
+	textconv = '/opt/conda/bin/python3' -m nbstripout -t
```

Comment:
Er dette standard installasjonspath for python3 på Mac? Hvis ikke, så endre til standard installasjonssti for python3.

Ellers bør fila døpes om til gitconfig-adm-mac, siden vi vel ikke har mac på citrix i prod-sona.